src/xenia/kernel/xam/xam_user.cc

```diff
@@ -816,6 +816,72 @@ dword_result_t XamProfileFindAccount_entry(
 }
 DECLARE_XAM_EXPORT1(XamProfileFindAccount, kUserProfiles, kImplemented);
 
+// https://github.com/TeaModz/XeLiveStealth-Full-Source/blob/d4a7439ac6241c4a13e883a6f156623d1c08f6eb/XeLive/Utils.cpp#L416
+dword_result_t XamUserLogon_entry(lpqword_t xuids_ptr, dword_t flags,
```

Comment:
It more likely looks like some action_id than flags, but I haven't checked it in XAM code yet